test/importer-flush.spec.js

```diff
@@ -14,7 +14,7 @@ const map = require('pull-stream/throughs/map')
 const collect = require('pull-stream/sinks/collect')
 const pushable = require('pull-pushable')
 
-describe('importer: flush', () => {
+describe.skip('importer: flush', () => {
```

Comment:
skip?

Reply:
There's no flushing any more as it only seemed to be called by the tests.  Have removed them now.